ultra/ultra/scenarios/common/distributions.py

```diff
@@ -1061,7 +1071,12 @@
                     "end": (150, 160),
                 },
             },
-        }
+        },
+        "ego_hijacking_params": {
+            "zone_range": [5, 10],
+            "wait_to_hijack_limit_s": 2,
+            "start_time": 3,  # any value or default for LANE_LENGTH / speed_m_per_s
```

Comment:
I noticed the `start_time` is different for these additions than the existing `ego_hijacking_params` for other distributions, is this okay?

Reply:
The `start_time` is a bit different for the intersection-specific scenarios. The other distribution do not require for the ego to spawn immediately thus `start_time` there is 10